tools/src/main/java/io/narayana/perf/Measurement.java

```diff
@@ -364,8 +364,6 @@ public Measurement<T> call() throws Exception {
         long start = System.nanoTime();
 
         try {
-            cyclicBarrier.await(); // wait for each thread to arrive at the barrier
```

Comment:
I think I noticed this code myself, do you think it is definitely wrong? It doesn't somehow let them all sync up in one place before the test starts in full or something?